website/pages/en/developing/substreams-powered-subgraphs-faq.mdx

```diff
@@ -0,0 +1,89 @@
+
+Developed by [StreamingFast](https://www.streamingfast.io/), Substreams is an exceptionally powerful processing engine capable of consuming rich streams of blockchain data. Substreams allow you to refine and shape blockchain data for fast and seamless digestion by end-user applications. More specifically, Substreams is a blockchain-agnostic, parallelized, and streaming-first engine, serving as a blockchain data transformation layer. Powered by the [Firehose](https://firehose.streamingfast.io), it ​​enables developers to write Rust modules, build upon community modules, provide extremely high-performance indexing, and [sink](/substreams/developers-guide/sink-targets/README/#substreams-sinks-overview) their data anywhere.
+
+Go to the [Substreams Documentation](/substreams/README/) to learn more about Substreams.
+
+## What are Substreams-powered subgraphs?
+
+[Substreams-powered subgraphs](/cookbook/substreams-powered-subgraphs/) combine the power of Substreams with the queryability of subgraphs. When publishing a Substreams-powered Subgraph, the data produced by the Substreams transformations, [formatted in a special way](https://github.com/streamingfast/substreams-sink-entity-changes/blob/develop/substreams-entity-change/src/tables.rs), are inserted into the database as if it was produced by the subgraph itself.
+
+If you are already familiar with subgraph development, then note that Substreams-powered subgraphs can then be queried, just as if it had been produced by the AssemblyScript transformation layer, with all the Subgraph benefits, like time-travel queries.
+
+## How are Substreams-powered subgraphs different from subgraphs?
+
+Both are used to index and query data from blockchains. They differ in how data transformations are handled. Subgraphs use AssemblyScript to allow you to model your data, whereas Substreams use smaller modules, coded in Rust. Both platforms target a WebAssembly virtual machine.
+
+Another major difference is that Substreams are massively parallelizable, whereas subgraphs need to run linearly across the blockchain history. This parallelization affects the programming paradigm, such that map and store modules are introduced in Substreams to help compute aggregations, whereas Postgres are used in subgraphs.
+
+## What are the benefits of using Substreams-powered subgraphs?
+
+Substreams-powered subgraphs combine all the benefits of Substreams with the queryability of subgraphs. They bring greater composability and high-performance indexing to The Graph. They also enable new data use cases; for example, once you've built your Substreams-powered Subgraph, you can reuse your [Substreams modules](/substreams/developers-guide/creating-your-manifest/#module-definitions) to output to different [sinks](/substreams/developers-guide/sink-targets/#substreams-sinks-overview) such as PostgreSQL, MongoDB, and Kafka.
+
+## What are the benefits of Substreams?
+
+There are many benefits to using Substreams, including:
```

Comment:
Access to additional data which is not available as part of the JSON RPC (this is maybe included in "all the benefits of firehose", but I think it is worth mentioning explicitly)